code/modules/surgery/surgery.dm

```diff
@@ -6,7 +6,7 @@
 	var/can_cancel = 1										//Can cancel this surgery after step 1 with cautery
 	var/list/species = list(/mob/living/carbon/human)		//Acceptable Species
 	var/location = "chest"									//Surgery location
-	var/requires_organic_bodypart = 1						//Prevents you from performing an operation on robotic limbs

```

Comment:
Unnecessary! There is a better way to do this!

Reply:
It actually makes the code a lot more modular. Instead of being either organic or not organic, it's whatever type is defined

Reply:
I was referring to the whole dm. Not just the reviewed area. I don't actually quite know how the review system works.